integration/cluster.go

```diff
@@ -58,10 +58,12 @@ import (
 )
 
 const (
-	tickDuration   = 10 * time.Millisecond
-	clusterName    = "etcd"
-	requestTimeout = 20 * time.Second
+	// time duration to wait for a request to go through or detect leader loss
```

Comment:
`// RequestWaitTimeout is the time duration...`?

Reply:
usually in go, we start the comment of an exported function/variable by state its name first.

Reply:
@fanminshi thank you!